lib/ParamedicAppUninstall.js

```diff
@@ -66,9 +66,14 @@ class ParamedicAppUninstall {
         return Q();
     }
 
-    uninstallAppIOS (targetObj, app) {
-        const uninstallCommand = 'xcrun simctl uninstall ' + targetObj.simId + ' uninstall ' + app;
```

Comment:
Are you sure this didn't work before?

Reply:
Yes, this didn't work, but it seem not to be actually a hard failure. It was failing on something else.

Here is example local test to see:
```
$ cordova create iosTest com.foobar.iosTest iosTest && $_
$ cordova platform add ios
$ cordova run ios --emulator --target "iPhone-XR, 12.2"
```
At this point you should see the app running.
```
$ instruments -s devices
...
iPhone Xʀ (12.2) [SIMULATOR UUID] (Simulator)
...
```
Get the UUID
```
$ xcrun simctl uninstall <UUID> uninstall com.foobar.iosTest
```
Notice the app still running and installed on simulator.
Update Command
```
$ xcrun simctl uninstall <UUID> com.foobar.iosTest
```
Notice the app is now not installed.